indico/modules/rb/controllers/backend/bookings.py

```diff
@@ -422,6 +423,31 @@ def _process(self):
         return jsonify(occurrence=reservation_occurrences_schema.dump(self.occurrence, many=False))
 
 
+class RHBookingOccurrenceLinkActions(RHBookingBase):
+    """Links a Reservation occurrence to an event."""
+
+    def _process_args(self):
+        RHBookingBase._process_args(self)
+        date = dateutil.parser.parse(request.view_args['date'], yearfirst=True).date()
+        self.occurrence = self.booking.occurrences.filter(ReservationOccurrence.date == date).one()
+        self.event = Event.query.filter_by(id=request.view_args['event_id'], is_deleted=False).first_or_404()
+
+    def _check_access(self):
```

Comment:
Shouldn't we also check for event management permissions here?

Reply:
I wasn't 100% sure about that, since you are not actually changing the event itself here 🤔

Still, I added the following:

```python
        if not self.event.can_manage(user):
            raise Forbidden('You cannot manage this event')
```

Reply:
Is that correct / sufficient?

Reply:
Yes, should be fine. And you're not changing the event itself but the booking will nonetheless start showing up in the event after linking it ;)

Reply:
Cool!